plugins/package-managers/node/src/main/kotlin/pnpm/Pnpm.kt

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright (C) 2019 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
```

Comment:
On a related note, I only skimmed other the code changes in this file due to a lack of time. As also the expected output changes, it's a bit hard to understand just from the diff whether the implementation is correct. However, I trust that you did the mentioned comparison to NPM results to get confidence that the new expected results are more correct than the old ones.

Reply:
pnpm list outputs a tree. Maybe this helpds with checking. Comparing with npm you can easily reproduce e.g. with a difftool comparing the two expected results.

Reply:
Above mentioned comparison should give a good idea that the change is highly likely correct (without verifying the change manually against the list output)